src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Cart/view.html.twig

```diff
@@ -34,19 +34,35 @@
   {% endblock %}
 
   <div class="row">
-    <div class="col">
-      {% block cart_customer_information %}
-        {% include '@PrestaShop/Admin/Sell/Order/Cart/Blocks/View/customer_information.html.twig' %}
-      {% endblock %}
-    </div>
-    <div class="col">
+    <div class="col-md-6">
       {% block cart_order_information %}
         {% include '@PrestaShop/Admin/Sell/Order/Cart/Blocks/View/order_information.html.twig' %}
       {% endblock %}
     </div>
+    <div class="col-md-6">
+      {% block cart_customer_information %}
+        {% include '@PrestaShop/Admin/Sell/Order/Cart/Blocks/View/customer_information.html.twig' %}
+      {% endblock %}
+    </div>
   </div>
 
   {% block cart_summary %}
     {% include '@PrestaShop/Admin/Sell/Order/Cart/Blocks/View/cart_summary.html.twig' %}
   {% endblock %}
+
+  {% if cartView.cartSummary.cart_link is not empty %}
+    <div class="card" data-role="customer-information">
+      <h3 class="card-header">
+        <i class="material-icons">mail</i>
+        {{ 'Share this cart'|trans({}, 'Admin.Orderscustomers.Feature') }}
```

Comment:
```suggestion
        {{ 'Share this cart'|trans({}, 'Admin.Orderscustomers.Feature') }}
```
What do you think about "Share checkout link"? If I understand well, the link will redirect the user to checkout page with the order details and invite them to submit payment?

Reply:
It's the same link that is used in abandoned cart modules and in backoffice order creation. When visiting it, it loads the cart in question to your browser along with all customer data. So you can just go to checkout.

Useful if you want to share it with a colleague or a customer. I could not find better name for this section.

Reply:
<img width="602" alt="Capture d’écran 2023-11-28 à 15 26 07" src="https://github.com/PrestaShop/PrestaShop/assets/43613217/b4205805-cf72-4499-807a-f6a837008467">
Here the wording used on the order page kinda of the same link

Reply:
@MatShir Yep, only difference is that I send him to cart instead of checkout here.